src/EditorFeatures/VisualBasic/ReferenceHighlighting/ReferenceHighlightingAdditionalReferenceProvider.vb

```diff
@@ -1,18 +0,0 @@
-' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-Imports System.Composition
-Imports System.Threading
-Imports System.Threading.Tasks
-Imports Microsoft.CodeAnalysis.Editor.Implementation.ReferenceHighlighting
-Imports Microsoft.CodeAnalysis.Host.Mef
-
-Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.ReferenceHighlighting
-    <ExportLanguageService(GetType(IReferenceHighlightingAdditionalReferenceProvider), LanguageNames.VisualBasic), [Shared]>
-    Friend Class ReferenceHighlightingAdditionalReferenceProvider
-        Implements IReferenceHighlightingAdditionalReferenceProvider
```

Comment:
IReferenceHighlightingAdditionalReferenceProvider was removed entirely.  It's not needed (will be clearer later).